addOns/pscanrulesBeta/src/test/java/org/zaproxy/zap/extension/pscanrulesBeta/SiteIsolationScanRuleTest.java

```diff
@@ -316,7 +317,7 @@ void shouldRaiseAlertGivenCoepHeaderIsMissing() throws Exception {
     }
 
     @Test
-    void shouldRaiseAlertGivenCoepHeaderIsNotEqualsToRequireCorp() throws Exception {
+    void shouldRaiseAlertGivenCoepHeaderIsNotExpectedValue() throws Exception {
         // Ref: https://html.spec.whatwg.org/multipage/origin.html#the-headers
```

Comment:
```suggestion
```
This a 301 and the fragment is outdated too. I'd suggest removing this since the scan rule itself already has references.